src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  background: center no-repeat url("images/star.svg");
```

Comment:
Add 'background-size: cover;' to ensure the star image fits the element properly.